Northstar.CustomServers/mod/scripts/vscripts/sh_progression.nut

```diff
@@ -84,6 +85,22 @@ bool function ClientCommand_SetProgression( entity player, array<string> args )
 
 	return true
 }
+
+bool function ClientCommand_ResetTitanAegis( entity player, array<string> args )
+{
+	if ( !args.len() )
+		return false
+	
+	int suitIndex = args[0].tointeger()
+	player.SetPersistentVar( "titanFDUnlockPoints[" + suitIndex + "]", 0 )
+	player.SetPersistentVar( "previousFDUnlockPoints[" + suitIndex + "]", 0 )
+	player.SetPersistentVar( "fdTitanXP[" + suitIndex + "]", 0 )
+	player.SetPersistentVar( "fdPreviousTitanXP[" + suitIndex + "]", 0 )
+	
+	RecalculateHighestTitanFDLevel( player )
```

Comment:
```suggestion
	
	// Refresh Highest Aegis Titan since we might get all of them back to 1 if players wants
	RecalculateHighestTitanFDLevel( player )
```
moving code comment, see other review comment